docs/ref/migration-operations.txt

```diff
@@ -195,6 +195,15 @@ if the field is nullable or if it has a default value that can be used to
 populate the recreated column. If the field is not nullable and does not have a
 default value, the operation is irreversible.
 
+.. admonition:: PostgreSQL
+
+    ``RemoveField`` specifies the ``CASCADE`` suffix to ensure `dependent
+    objects outside the table are also dropped`_. This will affect any
+    additional database objects created like views, etc.
```

Comment:
Hey @shangxiao, thank you for this branch!

Question/comment: when I first read this admonition, I thought the message meant that one could decide whether `CASCADE` is set or not, but after reading docs and seeing code, this is not optional, is a fact. Do you think we could rephrase this to be more explicit about it? What about something like:

```suggestion
    ``RemoveField`` will also delete any additional database objects that are
    related to the field being removed (like views for example). This is
    because the resulting SQL for PostgreSQL will use the ``CASCADE`` modifier
    to ensure `dependent objects outside the table are also dropped`_.
```

Reply:
> when I first read this admonition, I thought the message meant that one could decide whether CASCADE is set or not

Ah yep when I reread it today I myself saw that I wasn't clear this is something specified in the SQL statement and not a feature users can toggle 🙂

You're rephrasing is good, though I _believe_ "clause" is actually the correct term to use 🤔 (I had "suffix" & you've got "modifier")

Reply:
We should also clarify "DROP" somewhere like "because the resulting `DROP COLUMN` SQL statement" or something to that effect.

Reply:
Pushed a version I'm happy with, if you agree, we can land! :star2:

Reply:
Thanks so much, looks good!